tests/refarm_pagination/test_pagination.py

```diff
@@ -1 +1,67 @@
-# @todo #556:30m Create unit tests for pagination classes.
+from django.conf import settings
+from django.core import paginator
+from django.test import TestCase
+from django.http import Http404
+
+from refarm_pagination import pagination
+
+
+class Paginator(TestCase):
+
+    def test_page_404error(self):
+        with self.assertRaises(Http404):
+            pagination.Paginator([], 1).page(2)
+
+    def test_page_result(self):
+        self.assertIsInstance(
+            pagination.Paginator([], 1).page(1),
+            paginator.Page,
+        )
+
+
+class NeighborPage(TestCase):
+
+    def test_first_page_url(self):
+        """First page gives url without `page` query string param."""
+        base_url = 'url'
+        self.assertEqual(
+            base_url,
+            pagination.NeighborPage(1).url(base_url),
+        )
+
+    def test_rest_page_urls(self):
```

Comment:
term `rest` is relative to previous test. But if this test fail, this relation will be not clear.
I suggest `test_second_page_url`. You have good docstring to specify all other details